arch/arm/core/aarch32/isr_wrapper.S

```diff
@@ -162,6 +165,15 @@ _idle_state_cleared:
 #endif /* CONFIG_EXECUTION_BENCHMARKING */
 	blx r3		/* call ISR */
 
+#if defined(CONFIG_CPU_CORTEX_A) || defined(CONFIG_CPU_CORTEX_R)
```

Comment:
...and here

Reply:
@stephanosio this code block in executed only for Cortex-A,R: could you add a sentence explaining inline what this _eoi is doing?